include/deal.II/grid/manifold.h

```diff
@@ -1130,24 +1182,27 @@ namespace Manifolds
   get_default_quadrature(const MeshIteratorType &iterator,
                          const bool              with_laplace)
   {
-    const std::pair<std::vector<Point<MeshIteratorType::AccessorType::space_dimension> >,
-          std::vector<double> > points_and_weights = get_default_points_and_weights(iterator,
-                                                     with_laplace);
-    return Quadrature<MeshIteratorType::AccessorType::space_dimension>(points_and_weights.first,
-           points_and_weights.second);
+    const auto points_and_weights = get_default_points_and_weights(iterator, with_laplace);
+    return Quadrature<MeshIteratorType::AccessorType::space_dimension>
+    ({points_and_weights.first.begin(), points_and_weights.first.end()},
+    {points_and_weights.second.begin(), points_and_weights.second.end()});
```

Comment:
I'm pretty sure this calls the constructor that takes two `std::vector`s as arguments, but that's entirely not obvious. Would you mind making this cast explicit?

Reply:
I explicitly wrote out the constructor calls.